testdata/serviceconfig.yml

```diff
@@ -17,16 +17,12 @@ rules:
         labels:
           source: source.name | "unknown"
           target: target.name | "unknown"
-          service: api.name | "unknown"
```

Comment:
do we need to remove these?  leaving them as "unknown" seems somewhat reasonable. it would also likely save us from having to update the example dashboard (with promql).